src/pages/errors/ObsoleteEntryPage.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Message } from 'franklin-sites';
+import { Link } from 'react-router-dom';
+import { v1 } from 'uuid';
+import { InactiveEntryReason, InactiveReasonType } from '../../model/uniprotkb/UniProtkbConverter';
+import ArtWork from '../../svg/obsolete-entry.svg';
+
+import '../../styles/ErrorPages.scss';
+
+type ObsoleteEntryPageProps = {
+  accession: string;
+  details: InactiveEntryReason;
+}
+
+const DeletedEntryMessage: React.FC<{accession: string}> = ({
+  accession,
+}) => (
+  <Message level="info">
+    <h4>This entry is obsolete</h4>
+    <p data-testid="deleted-entry-message">
+      The protein sequence for this entry is available in {' '}
+      <Link to={`/uniparc/?query=${accession}`}>UniParc</Link>.
+      For previous versions of this entry, please look at its {' '}
+      <Link to={`/uniprot/${accession}?version=*`}>history</Link>.
+    </p>
+  </Message>
+);
+
+const DemergedEntryMessage: React.FC<{accession: string, demergedTo: string[]}> = ({
+  accession,
+  demergedTo,
+}) => (
+  <Message level="info">
+    <h4>This entry is obsolete</h4>
+    <p data-testid="demerged-entry-message">
+      It can now be found as secondary accession in {' '}
+      {demergedTo
+        .reduce((a: (string | JSX.Element)[], c, i) => {
+          if (i > 0) {
+            if (i === demergedTo.length - 1) {
+              a.push(<span key={v1()}> and </span>);
```

Comment:
Let's just stop using uuids for keys shall we 😅.
Maybe the string value, or the even here I'd be OK using the index

Reply:
Oh, I should have used  a simple ` and ` string there! But I agree with you, I don't like using the `uuid` neither. I think when I did it there I had a different logic there and I did initially compose a key like `${c}-and`, but thought using `uuid` might be cleaner, specially because this is not an element which would get any updates.

Maybe we should come up with a few rules-of-thumb to know when to use `uuid` and when shouldn't. I think there are cases that it can be useful, so I'd vote against banning it completely.

Reply:
I mean, that shouldn't ever be used for React keys. Because across render they would change so trigger an update. At least here it's not dynamic content so using index should be fine enough.
Apart of that, outside of React keys, yes, using uuid wherever needed is fine.

Reply:
I think react is going to complain if you ever use any numerical `index` value as the key, but I will look into it. I agree the extra render, however minor, should not happen if it can be prevented easily.

Reply:
I think the only time React complains would be duplicate keys (and by the way I already so that warning somewhere, I'll need to investigate it next time I see it). If there is any complaint about index, that would be a linting rule that would be easy enough to override locally.

Reply:
Yeah, you might be right. From React docs: `When you don’t have stable IDs for rendered items, you may use the item index as a key as a last resort`.

Reply:
Addressed this too.